### PR TITLE
Removes most melee mining tools. Pickaxes still included.

### PR DESCRIPTION
**Pull request by nemvar**
## About The Pull Request

Removes all non cyborg drills from the game.
Also removes the diamond and silver pickaxe.
Certain loot tables have been adjusted to fit this change. Decaboxes now have a chance to drop gold and necro chests have a higher chance to drop certain modkits. (lol you thought I would buff something) 
Buffs the speed at which shovels gather sand.

## Why It's Good For The Game

The drills are just kind there and not doing much.

## Changelog
:cl:
balance: Removed a bunch of unused mining tools. This includes the mineral pickaxes, the handdrills and the jackhammer.
balance: Shovels now gather sand faster.
/:cl:

### CODE REVIEW

**Comment by wesoda25** [comment]
make it remove walls at half the speed it would take to deconstruct them

**Comment by nemvar** [comment]
No.

**Comment by Dawson1917** [comment]
Even with mining rocks _literally instantly_ the diamond jackhammer is less useful than the plasma cutter, since the advanced cutter already mines as fast as you run, and miners building jackhammers is going to absolutely deprive the station of materials even more than it currently does as it costs 5 diamond sheets to make one

jackhammer grief is real af but i very much enjoy using them to smash down science when they've closed the shutters and refused to do anything in the shift, or for away missions or for space, or for cult bases and malf AI cores during summoning/delta. It really needs some kind of other purpose that isn't wall grief since it's still worthless for mining

> 
> 
> make it remove walls at half the speed it would take to deconstruct them

doesn't solve the problem

**Comment by nemvar** [comment]
Well, mining rework is supposedly coming, so the buff doesn't really matter. Tbh, we could just remove all melee mining tools (pickaxes, drills, resonator, jackhammer) and barely anyone would notice except that you don't get diamond pickaxes from necro crates anymore.

**Comment by Dawson1917** [comment]
The jackhammer and other melee mining tools sans golem resonators and public lavaland pickaxes may as well be removed, yeah. Unless someone thinks of a creative use for the jackhammer (I can't come up with any) it'll probably just be a noob trap and resource eater after removing its ability to delete walls. Unless you are for some reason a hulk or golem and can't use plasma cutters to mine.

**Comment by Tlaltecuhtli** [comment]
remove it

**Comment by InhumanPwnage** [comment]
Jackhammer is great for breaking walls yea, and obviously mining glass/rocks instantly. Its concept is great, and best used against malf AI, although if miners were the only hope agaisnt malf AI (like in lowpop rounds), gibtonite is to be used instead, except, it will take some time to find some (say 3-4) and cautiously stop reactions in the during the 388 second nuclear countdown, especially if the AI gets borgs to defend itself. 

Granted, this reason alone doesn't stop the jackhammer from being misused. I do think it's fair currently that it's a huge item and has to be visibly shown in hand until BoHs (or held on your belt).

My only other suggestion for the jackhammer to have a use, is an idea I had where rocks on lavaland are layered - say 3 layers, soft layer (what we have right now), hard layer (plasma cutters have halved effect), and obsidian (PKA needs to be x3 damage, plasma cutter only breaks 1 rock, while jackhammer breaks the obsidian at normal speed, making it of use for this layer). Buuuut i think this suggestion is best put up for discussion elsewhere.

**Comment by Whoneedspacee** [comment]
Absolutely worthless buff for a few reasons.

-Still slower than plasma cutters which are still cheaper, you can’t make the jackhammer better without giving it something unique.

-Even worse than plasma cutters now since they can at least speedily weld down walls.

-Can’t scout the area ahead, have fun blowing yourself up with gibtonite.

This pretty much just makes the weapon worthless and I don’t even see it used that much anyways, what brought on this change?

**Comment by Whoneedspacee** [comment]
tl;dr This is a massive nerf with no buffs, and you state yourself that it rewards “risky and skillful gameplay”.

How could it do that if you randomly run into gibtonite and (per the ores wording) lose a couple limbs.

There would be absolutely no reward to using this item.

**Comment by Whoneedspacee** [comment]
I think it’d be interesting though if the jackhammer had increased ore output. That might encourage the type of playstyle that you’re going for.

**Comment by nemvar** [comment]
Yes, it is a massive nerf with no real buffs. Deal with it.

**Comment by Whoneedspacee** [comment]
>nerf tool that's already barely used into the ground

can we just remove all the melee tools then to be honest like the other dude said.

**Comment by bgobandit** [comment]
possibly dumb question but how does this affect ash lizards

**Comment by nemvar** [comment]
Ashlizards don't get drills. Pickaxes are still in the game.

**Comment by kriskog** [comment]
Jackhammers and drills are good for golems, since they can't use cutters. This leaves their options at basically only a modded pka, which is a bit sad.

**Comment by nemvar** [comment]
You still have the resonator anon.

**Comment by kriskog** [comment]
Well I was talking about mining upgrades, not the roundstart resonator :eyes:

**Comment by nemvar** [comment]
Wasn't upgraded resonator a thing?

**Comment by kriskog** [comment]
Sure, there is one. Its just not very good. Or really distinguishably better for mining than the standard one.

**Comment by CRITAWAKETS** [comment]
the resonators are absolute trash

**Comment by kriskog** [comment]
Gonna be honest, I only ever used the jackhammer or pka as golem. The rest are trash. I dont disagree with removing the wall thing, or the shit pickaxes and drills, but I do wish we at least had one melee mining upgrade thats worth using. How about buff one of the drills for mining and leave that in?

**Comment by nemvar** [comment]
Alright, gonna buff the resonators.

**Comment by Krysonism** [comment]
But drills are cool nemvar. Can't we let them drill down windows or something?

**Comment by Cyprex** [comment]
Leave the jackhammer in for golems, resonators are total trash and unfun to use.

**Comment by nemvar** [comment]
If anything dies today, it's the jackhammer. It's the only reason why I even care enough to do this.

**Comment by YakumoChen** [comment]
Keep the diamond pick. Pacifists need something better than a resonator to mine and work to upgrade.

**Comment by YakumoChen** [comment]
You'd also be affecting drop tables for tendril chests and taking it out of the mining vendor if you remove diamond pick.

**Comment by lolman360** [comment]
free golems can use jackhammer to mine about as fast as a plasmacutter. why you do this.

**Comment by lolman360** [comment]
wait golems have triggerguard+aoemining

**Comment by YakumoChen** [comment]
My biggest concern is that there won't be a better option for mining if the research is simply unable to be done for whatever reason. The KPA is not really that useful as an actual mining tool, neither is the resonator.

**Comment by cacogen** [comment]
Can you get rid of the shitty, default pickaxe and replace it with one of the better tools? I hate spawning as a Lavaland mob only to spend 15 minutes trying to dig out of there.

**Comment by TerraGS** [comment]
I'm fine with removing them since my experiences with drill/jackhammer has often been immediately dying because I auto-mined into gibtonite.

**Comment by ishitbyabullet** [comment]
Why are you removing pickaxes? Now every role that could mine their way out in the past is gonna need a powerful weapon. Escape pods will now need a powerful weapon. Ashwalkers need a powerful weapon now.

It also just seems stupid and means that everyone, from backwards savages who use spears and stones to the highly advanced and futuristic explorers of space, use the same fucking tools to mine. That doesn't sound right at all.

**Comment by actioninja** [comment]
> Why are you removing pickaxes? Now every role that could mine their way out in the past is gonna need a powerful weapon. Escape pods will now need a powerful weapon. Ashwalkers need a new powerful weapon now. While pickaxes are good melee weapons themselves, I think resonators can deal more damage if you actually know how to use it. God forbid if they get handed KpAs though.
> 
> It also just seems stupid and means that everyone, from backwards savages who use spears and stones to the highly advanced and futuristic explorers of space, use the same fucking tools to mine. That doesn't sound right at all.

he's not removing pickaxes.

**Comment by ishitbyabullet** [comment]
> he's not removing pickaxes.

Ah, alright.

**Comment by Dorsisdwarf** [comment]
Plasma cutters were a mistake and should have been removed instead of being nerfed back to mining tools (that were still better than every other mining tool on lavaland).

**Comment by nemvar** [comment]
Floyd told me to close this. Very sad.

**Comment by adamkad1** [comment]
did you think about golems when making this?
